frontend/src/components/Reports/Routine.js

```diff
@@ -123,8 +123,8 @@ export const RoutineReportsMenu = {
       icon: IbmWatsonNaturalLanguageUnderstanding,
       SideNavMenuItem: [
         {
-          link: config.serverBaseUrl + "/AuditTrailReport",
-          label: <FormattedMessage id="sideNav.label.audittrail" />,
+          link: "/AuditTrailReport?type=routine",
```

Comment:
use    `link: "/Routinetype=routine", and render the report from [here  ](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop_3x/frontend/src/components/Reports/routine/Index.js#L44-L104)